SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs

```diff
@@ -576,45 +576,28 @@ public UpdateAllocationResponse UpdateAllocation(UpdateAllocationRequest request
         #region Warning Notes
         public PostWarningNoteResponse PostWarningNote(PostWarningNoteRequest request)
         {
-            Person person = _databaseContext.Persons.FirstOrDefault(x => x.Id == request.PersonId);
+            var person = _databaseContext.Persons.FirstOrDefault(x => x.Id == request.PersonId);
 
             if (person == null)
             {
                 throw new PostWarningNoteException($"Person with given id ({request.PersonId}) not found");
             }
 
-            //TODO: Extract request to domain process to EntityFactory
-            WarningNote warningNote = new WarningNote()
-            {
-                PersonId = request.PersonId,
-                StartDate = request.StartDate,
-                EndDate = request.EndDate,
-                DisclosedWithIndividual = request.DisclosedWithIndividual,
-                DisclosedDetails = request.DisclosedDetails,
-                Notes = request.Notes,
-                NoteType = request.NoteType,
-                Status = "open",
-                DisclosedDate = request.DisclosedDate,
-                DisclosedHow = request.DisclosedHow,
-                WarningNarrative = request.WarningNarrative,
-                ManagerName = request.ManagerName,
-                DiscussedWithManagerDate = request.DiscussedWithManagerDate,
-                CreatedBy = request.CreatedBy
-            };
+            var warningNote = request.ToDatabaseEntity();
 
             _databaseContext.WarningNotes.Add(warningNote);
             _databaseContext.SaveChanges();
 
-            PostWarningNoteResponse response = new PostWarningNoteResponse()
+            var response = new PostWarningNoteResponse
             {
                 WarningNoteId = warningNote.Id
             };
 
             // try
```

Comment:
Could we just delete the odd bit of commented out try code and same for the catch block at the bottom.

Reply:
Yeah, I wasn't sure if this was left as a reminder since there's a [commented out test](https://github.com/LBHackney-IT/social-care-case-viewer-api/blob/cfc755d67925c372f5c77373778658d94847630c/SocialCareCaseViewerApi.Tests/V1/Gateways/DatabaseGateway.Tests.cs#L513) in database gateway tests that could be linked to this 🤔